src/pivotal-ui/components/forms/stepper.js

```diff
@@ -0,0 +1,61 @@
+var FormStepper = function(el, input, decrementButton, incrementButton){
+  this.stepper         = $(el);
+  this.input           = input || this.stepper.find("input[type=number]");
+  this.decrementButton = decrementButton || this.stepper.find(".btn-decrement");
+  this.incrementButton = incrementButton || this.stepper.find(".btn-increment");
+
+  this.min = parseInt(this.input.attr("min")) || 0;
+  this.max = parseInt(this.input.attr("max")) || 99999;
+  this.value = parseInt(this.input.val()) || parseInt(this.min);
+};
+
+FormStepper.prototype.increment = function(){
+  var val = parseInt(this.value) || this.min;
```

Comment:
leading zero bugs in `parseInt` without radix?

Also will throw, in response to user input.

Consider using `Number()` instead?